serverless-al2-example/serverless.yml

```diff
@@ -0,0 +1,54 @@
+service: example-aws-golang-with-al2-dd-bridge
+
+frameworkVersion: "3"
+
+provider:
+  name: aws
+  runtime: provided.al2
+  # architecture: arm64 # use if you want to run Lambda Graviton ARM-based
+  architecture: x86_64
+
+functions:
+  bootstrap:
+    handler: hello-src/main.go
+    layers:
+      - arn:aws:lambda:us-east-1:666523192759:layer:codesee-dd-bridge:6 # This version currently only support x86, no ARM support yet.
+    environment:
+      DD_APM_DD_URL: http://127.0.0.1:8080
+      DD_APM_NON_LOCAL_TRAFFIC: true
+      DD_APM_ENABLED: true
+
+      CODESEE_ENABLE_ACCESS_LOGS: true # DEBUG - turn off in Prod
```

Comment:
I doubt everyone will read things carefully. Maybe we set this to `false` but leave a comment to "turn on for debug logging for codesee layer"

Reply:
will flip around